### DISCUSSION TITLE
Add support for custom model data in CMI (worth.yml & item checks)

### DISCUSSION THREAD

**Discussion by lerndmina** [Zrips/CMI#6837] [Enhancement]
It'd be great to be able to add items to the worth.yml that use custom model data so when checking against that file the system will differentiate between 2 items of the same type based on custom model data.

This feature would be really helpful when making hologram shops that would check the items worth based on what is in the player's hand.

**Comment by mrfloris** [comment]
If we use /cmi setworth in-game and it detects it has custom data number on it, i wouldn't mind if it gets appended to worthy.yml with a cd: prefix, so we can also easily search for them in the file and remove old items that sometimes get an update through other plugins. that way we dont end up with 

diamond:price:something
diamond:price:somethingcustom1
diamond:price:somethingcustom2
etc
i rather search for cd:diamond.

^^^ just me thinking outloud for my personal use.

**Comment by mrfloris** [comment]
Check the latest cmi build 9.1.4.2, it has this in the changelog:

```
Now setworth supports custom model data. If price is not set for specific custom model data then we will default to one which doesn't have any set, if possible. While setting item price with custom model data it will be indicated with id number in a brackets in UI's title.
```
